Documentation/proposals/20250122-blackbox-crd.md

```diff
@@ -40,9 +40,9 @@ The Probe CRD provides a partial solution for handling complex modules like HTTP
 
 ## How
 
-As discussed in [Discussion](https://github.com/prometheus-operator/prometheus-operator/discussions/7134#discussioncomment-11569949), we propose creating two new Custom Resource Definitions (CRDs):
+We propose creating two new Custom Resource Definitions (CRDs):
 - The BlackboxExporter CRD, which functions similarly to the Prometheus CRD, will manage the deployment configuration and lifecycle of blackbox-exporter instances
-- The BlackboxExporterModule CRD will define module specifications and automatically generate blackbox configurations, analogous to how ScrapeConfig works in Prometheus Operator
+- The BlackboxExporterModule CRD will define module specifications and automatically generate blackbox configurations, analogous to how Alertmanager works in Prometheus Operator
```

Comment:
Might be a stupid question. But isn't generating a configuration based on the CRD more relevant to the `ScrapeConfig` - `Prometheus` relationship?

Reply:
Well IMO considering how BB works and it needs the prober with the prober type and relevant configuration for each probe I think is more a relation if configuration, so I think is more closer related to Alertmanager|AlermanagerConfig